src/m365/graph/commands/openextension/openextension-set.ts

```diff
@@ -0,0 +1,129 @@
+import { z } from 'zod';
+import { globalOptionsZod } from '../../../../Command.js';
+import { zod } from '../../../../utils/zod.js';
+import { validation } from '../../../../utils/validation.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import { Logger } from '../../../../cli/Logger.js';
+import commands from '../../commands.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { Extension } from '@microsoft/microsoft-graph-types';
+import { optionsUtils } from '../../../../utils/optionsUtils.js';
+
+const options = globalOptionsZod
+  .extend({
+    name: zod.alias('n', z.string()),
+    resourceId: zod.alias('i', z.string()),
+    resourceType: zod.alias('t', z.enum(['user', 'group', 'device', 'organization'])),
+    keepUnchangedProperties: zod.alias('r', z.boolean().optional())
```

Comment:
```suggestion
    keepUnchangedProperties: zod.alias('k', z.boolean().optional())
```

The short option is not really logical here. What does 'r' stand for? We could use 'k' instead